Documentation/ApiOverview/MessageBus/Index.rst

```diff
@@ -220,6 +220,32 @@ The TYPO3 Core has been tested with three transports:
 *   :php:`\Symfony\Component\Messenger\Transport\InMemory\InMemoryTransport`
     (for testing)
 
+Add rate limiter
+----------------
+
+..  versionadded:: 13.4
+    You can add your own rate limiter definition to asynchronous messages
+
+Rate limiting can be applied to asynchronous messages processed through the
+consume command. This allows controlling message processing rates to:
+
+*   Stay within external service limits (API quotas, mail sending thresholds)
+*   Manage server resource utilization
+
+Example
```

Comment:
```suggestion
Example: Usage of a rate limiter
```

Headlines should not be generic but tell ppl what to expect in that section. This also improves search results

Reply:
Was a copy&paste from another file